### PR TITLE
Updating the PostGallery migration script.

### PR DESCRIPTION
**Pull request by weerd**
### What does this PR do?

This PR simply updates the PostGallery migration script to include some of the recent fields added and other changes.

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: contentful/content-types/postGallery.js [hunk 1]
```diff
@@ -52,21 +76,24 @@ module.exports = function(migration) {
...
     .disabled(false)
     .omitted(false);
-
   postGallery.changeEditorInterface('internalTitle', 'singleLine', {});
 
   postGallery.changeEditorInterface('actionIds', 'listInput', {
-    helpText: `A comma-separated list of one or more Action IDs to display in this gallery. Action IDs
-    can be found in Rogue: https://dosome.click/nyshrf`,
+    helpText:
+      'A comma-separated list of one or more Action IDs to display in this gallery. Action IDs\n    can be found in Rogue: https://dosome.click/nyshrf',
```
Comment (DFurnes) [inline contentful/content-types/postGallery.js]: I don't think we actually need a `\n` here, I'd just used a template string for readability.

Comment (weerd) [reply contentful/content-types/postGallery.js]: I didn't add it. It's from the generated migration. I figured Contentful's generation script knows best 🤷‍♂️

Comment (weerd) [reply contentful/content-types/postGallery.js]: It's likely going to keep showing up every time we pull it down, so I figured _why fight it...

Comment (DFurnes) [reply contentful/content-types/postGallery.js]: Ohhhhhhhh! I had no clue you could even do that! 😲

**Pull request merged**